src/processors/EmbraceSessionBatchedSpanProcessor/EmbraceSessionBatchedSpanProcessor.ts

```diff
@@ -9,22 +9,35 @@ import { EMB_TYPES, KEY_EMB_TYPE } from '../../constants/index.js';
 import type { SessionSpan } from '../../instrumentations/index.js';
 import { EmbraceProcessor } from '../EmbraceProcessor/index.js';
 import type { EmbraceSessionBatchedSpanProcessorArgs } from './types.js';
-import type { LimitManagerInternal } from '../../managers/index.js';
+import type {
+  LimitManagerInternal,
+  SpanSessionManagerInternal,
+} from '../../managers/index.js';
 import { EmbraceSpanStorage } from '../../utils/index.js';
 
 const isSessionSpan = (span: ReadableSpan | SessionSpan): span is SessionSpan =>
   span.attributes[KEY_EMB_TYPE] === EMB_TYPES.Session;
 
+type ExportFailureReason = 'concurrent_limit' | 'fetch_error' | 'unknown';
+
+const exportFailureAttributeKey = (
+  reason: ExportFailureReason,
+  session: 'current' | 'previous'
+) =>
+  `emb.${session === 'current' ? 'export_failed' : 'previous_export_failed'}.${reason}`;
```

Comment:
I would probably change this so that reason is the value instead of a different key.
e.g.:
`emb.export_failure = fetch_error` / `emb.previous_export_failure = fetch_error`

thoughts?